README.md

```diff
@@ -35,33 +29,109 @@ Open VSCode. Go to extensions and search for "ObjectScript" like it is shown on
 Or install from ObjectScript extension page on [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=daimor.vscode-objectscript)
 ![installation](https://raw.githubusercontent.com/intersystems-community/vscode-objectscript/master/images/installation.gif)
 
-## Configure connection
+## Configure Connection
+
+To be able to use many features you first need to configure the connection to your IRIS/Caché/Ensemble server(s) in your [VSCode settings](https://code.visualstudio.com/docs/getstarted/settings). If you are unfamiliar with how settings work and how they are edited, use that link.
+
+We recommend you define server connections in the `intersystems.servers` object whose structure is defined by the [InterSystems Server Manager](https://marketplace.visualstudio.com/items?itemName=intersystems-community.servermanager) helper extension. Install this extension to get assistance when editing the JSON definition. For example:
```

Comment:
You type this manually into your configuration file -- settings.json or your [.code-workspace file](https://code.visualstudio.com/docs/editor/multi-root-workspaces#_workspace-file) -- we recommend you define server connections in ...

Reply:
@gjsjohnmurray can you update the above to add this subtlety?